react-client/src/components/PlayerEnterNameScreen.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class WelcomeScreen extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    return (
+      <div>
+
+        <h2> What should the Heading Be? </h2>
+
+
+      I do not presently know how to deal with the html form
+
+     
+<form >
+  <input type="text" name="Token" value="Enter Access Token"/>
+
```

Comment:
there should only be one submit button for both token and name otherwise the user may expect that after submitting the token they would be taken to another page where they enter their name